music21/voiceLeading.py

```diff
@@ -769,6 +814,63 @@ def hiddenOctave(self):
         '''
         return self.hiddenInterval(self.octave)
 
+    def voiceOverlap(self):
+        '''
+        Returns True if the second note in V1 is lower than the first in V2, or
+        if the second note in V2 is higher than the first note in V1.
+
+        We will make the examples shorter with this abbreviation:
+        >>> N = note.Note
+
+        >>> vl = voiceLeading.VoiceLeadingQuartet(N('A4'), N('B4'), N('F4'), N('G4'))
+        >>> vl.voiceOverlap() # no overlap
+        False
+
+        >>> vl = voiceLeading.VoiceLeadingQuartet(N('A4'), N('B4'), N('F4'), N('A4'))
+        >>> vl.voiceOverlap() # Motion to the SAME note is not considered overlap
+        False
+
+        >>> vl = voiceLeading.VoiceLeadingQuartet(N('A4'), N('C4'), N('F4'), N('Bb4'))
+        >>> vl.voiceOverlap() # V2 overlaps V1
+        True
+
+        >>> vl = voiceLeading.VoiceLeadingQuartet(N('A4'), N('E4'), N('F4'), N('D4'))
+        >>> vl.voiceOverlap() # V1 overlaps V2
+        True
+        '''
+        if self.v1n2.pitch < self.v2n1.pitch or self.v2n2.pitch > self.v1n1.pitch:
```

Comment:
This assumes that v1 is higher than v2 -- perhaps we should make that a requirement?